elements/bulbs-clickventure/bulbs-clickventure.js

```diff
@@ -18,6 +19,9 @@ class BulbsClickventure extends BulbsHTMLElement {
     let $element = $(this);
     $element.data('clickventurePlugin', new Clickventure($element));
     $element.on('clickventure-page-change', fireAnalytics);
+
+    // Embeded CVs can resize their parent frame
+    resizeParentFrame(document.getElementsByTagName('body')[0].clientHeight);
```

Comment:
Looks nice, can use the shortcut `document.body` here to get the body element.

Reply:
Thank you kind sir!